hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieClusteringConfig.java

```diff
@@ -182,6 +182,12 @@ public class HoodieClusteringConfig extends HoodieConfig {
       .sinceVersion("0.7.0")
       .withDocumentation("Each group can produce 'N' (CLUSTERING_MAX_GROUP_SIZE/CLUSTERING_TARGET_FILE_SIZE) output file groups");
 
+  public static final ConfigProperty<Boolean> PLAN_STRATEGY_FORCE = ConfigProperty
+      .key(CLUSTERING_STRATEGY_PARAM_PREFIX + "force")
+      .defaultValue(true)
```

Comment:
IMO, the default value of this configuration should be false.

Reply:
IF the default value is false，too many ut need to modify to make it right,